README.md

```diff
@@ -20,103 +20,45 @@ You can either use environment variables or flags to configure the following set
+brew install kubernetes-helm
+helm init --history-max 25 --upgrade
+helm package chart/estafette-gke-node-pool-shifter --version 1.0.11
+helm upgrade estafette-gke-node-pool-shifter estafette-gke-node-pool-shifter-1.0.11.tgz --namespace estafette --install --set rbac.create=true --set googleServiceAccount=$(./google_service_account.json | base64)
 ```
 
-See [documentation](https://developers.google.com/identity/protocols/application-default-credentials).
-
-After what you also have to deploy the rbac.yaml file which set role and permissions inside the cluster. Then deploy
-the application to Kubernetes cluster using the manifest below.
-
-
-```yaml
-apiVersion: v1
-kind: Namespace
-metadata:
-  name: estafette
----
-apiVersion: v1
-kind: ServiceAccount
-metadata:
-  name: estafette-gke-node-pool-shifter
-  namespace: estafette
-  labels:
-    app: estafette-gke-node-pool-shifter
----
-apiVersion: v1
-kind: Secret
-metadata:
-  name: estafette-gke-node-pool-shifter-secrets
-  namespace: estafette
-  labels:
-    app: estafette-gke-node-pool-shifter
-type: Opaque
-data:
-  google-service-account.json: ${GOOGLE_SERVICE_ACCOUNT}
----
-apiVersion: extensions/v1beta1
-kind: Deployment
-metadata:
-  name: estafette-gke-node-pool-shifter
-  namespace: estafette
-  labels:
-    app: estafette-gke-node-pool-shifter
-spec:
-  replicas: 1
-  strategy:
-    type: Recreate
-  selector:
-    matchLabels:
-      app: estafette-gke-node-pool-shifter
-  template:
-    metadata:
-      labels:
-        app: estafette-gke-node-pool-shifter
-    spec:
-      serviceAccount: estafette-gke-node-pool-shifter
-      terminationGracePeriodSeconds: 300
-      containers:
-      - name: estafette-gke-node-pool-shifter
-        image: estafette/estafette-gke-node-pool-shifter:latest
-        ports:
-        - name: prom-metrics
-          containerPort: 9001
-        env:
-        - name: NODE_POOL_FROM
-          value: default-pool
-        - name: NODE_POOL_TO
-          value: preemptible-pool
-        - name: GOOGLE_APPLICATION_CREDENTIALS
-          value: /etc/app-secrets/google-service-account.json
-        resources:
-          requests:
-            cpu: 10m
-            memory: 16Mi
-          limits:
-            cpu: 50m
-            memory: 128Mi
-        livenessProbe:
-          httpGet:
-            path: /metrics
-            port: prom-metrics
-          initialDelaySeconds: 30
-          timeoutSeconds: 1
-        volumeMounts:
-        - name: app-secrets
-          mountPath: /etc/app-secrets
-      volumes:
-      - name: app-secrets
-        secret:
-          secretName: estafette-gke-node-pool-shifter-secrets
+### Deploy without Helm
+
+```
+export NAMESPACE=estafette
+export APP_NAME=estafette-gke-node-pool-shifter
+export TEAM_NAME=tooling
+export VERSION=1.0.11
+export GO_PIPELINE_LABEL=1.0.11
+export GOOGLE_SERVICE_ACCOUNT=$(cat google-service-account.json | base64 -w 0)
```

Comment:
`-w 0` is needed on linux, but doesn't work on Mac. Default on Mac is single line already.